libs/shared/devops-copilot/feature/src/lib/devops-copilot-panel/use-thread.tsx

```diff
@@ -0,0 +1,84 @@
+import { useAuth0 } from '@auth0/auth0-react'
+import { useEffect, useState } from 'react'
+import { type Thread } from './devops-copilot-panel'
+import { HACKATHON_API_BASE_URL } from './submit-message'
```

Comment:
This should be coming from an env variable and shouldn't be imported from the neighbor component